### PR TITLE
Improvised for better

### PR DESCRIPTION
**Pull request by RockingSNP**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers
Tags added

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4700"></a>

### CODE REVIEW

File: data/RockingSNP.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+{
+      "group": "Community"
+  "name": "Join our Enthusiastic Open-source Napptive Community",
+      "url": "https://join.slack.com/t/napptivecommunity/shared_invite/zt-1ov6ocst2-BsZDhfOZYAet86D6TxlIkg",
+      "icon": "FaLink"
+    },
+{
+      "group": "Community"
+  "name": "Join our Enthusiastic Open-source GrowInCommunity",
+      "url": "https://join.slack.com/t/napptivecommunity/shared_invite/zt-1ov6ocst2-BsZDhfOZYAet86D6TxlIkg",
```
Comment (priyankarpal) [inline data/RockingSNP.json:68]: I think you added the wrong link. Please add a relevant link.
``` suggestion
      "URL": "Add-Your-Link",
```

File: data/RockingSNP.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      "icon": "FaLink"
+    },
```
Comment (priyankarpal) [inline data/RockingSNP.json:70]: Delete the last comma 
```suggestion
    }
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/RockingSNP.json (hunk 1 above)
Comment (RockingSNP) [reply data/RockingSNP.json]: hmm

**Review by priyankarpal — CHANGES_REQUESTED** [review]
hey, @RockingSNP this branch has a conflict error. Please resolve it. You can check the official doc about conflict error https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github

**Comment by RockingSNP** [comment]
> hey, @RockingSNP this branch has a conflict error. Please resolve it. You can check the official doc about conflict error https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github

I've Resolved it....

File: data/RockingSNP.json [hunk 3]
```diff
@@ -1,20 +1,75 @@
...
+  ],
+  
+  "links": [
     {
       "name": "Follow me on Twitter",
       "url": "https://twitter.com/SouvikP74860962",
       "icon": "FaTwitter"
     },
     {
+      "group": "Socials"
```
Comment (priyankarpal) [inline data/RockingSNP.json]: Add a comma  
```suggestion
      "group": "Socials",
```

File: data/RockingSNP.json [hunk 4]
```diff
@@ -1,20 +1,75 @@
...
       "name": "Follow me on Github",
       "url": "https://www.github.com/RockingSNP/",
       "icon": "FaGithub"
     },
   {
+       "group": "Socials"
```
Comment (priyankarpal) [inline data/RockingSNP.json]: Add a comma
```suggestion
       "group": "Socials",
```

File: data/RockingSNP.json [hunk 5]
```diff
@@ -1,20 +1,75 @@
...
+       "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/souvik-p-117647218/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "group": "Socials"
```
Comment (priyankarpal) [inline data/RockingSNP.json]: Add a comma
```suggestion
      "group": "Socials",
```

File: data/RockingSNP.json [hunk 6]
```diff
@@ -1,20 +1,75 @@
...
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/rockingsp_official/",
+      "icon": "FaInstagram"
+    },
+   {
+      "group": "Assignments"
```
Comment (priyankarpal) [inline data/RockingSNP.json]: Add a comma
```suggestion
      "group": "Assignments",
```

File: data/RockingSNP.json [hunk 7]
```diff
@@ -1,20 +1,75 @@
...
+      "name": "Checkout my works on Replit",
+      "url": "https://replit.com/@RockingSNP",
+      "icon": "SiReplit"
+    },
+    {
+      "group": "Community"
```
Comment (priyankarpal) [inline data/RockingSNP.json]: Add a comma
```suggestion
      "group": "Community",
```

File: data/RockingSNP.json [hunk 8]
```diff
@@ -1,20 +1,75 @@
...
+      "name": "Get into our Enthusiastic Open-source EddieHub Community",
+      "url": "http://eddiehub.org",
+      "icon": "FaLink"
+    },
+{
+      "group": "Community"
```
Comment (priyankarpal) [inline data/RockingSNP.json]: Add a comma

```suggestion
      "group": "Community",
```

File: data/RockingSNP.json [hunk 9]
```diff
@@ -1,20 +1,75 @@
...
+  "name": "Join our Enthusiastic Open-source Napptive Community",
+      "url": "https://join.slack.com/t/napptivecommunity/shared_invite/zt-1ov6ocst2-BsZDhfOZYAet86D6TxlIkg",
+      "icon": "FaLink"
+    },
+{
+      "group": "Community"
```
Comment (priyankarpal) [inline data/RockingSNP.json]: Add a comma
```suggestion
      "group": "Community",
```

File: data/RockingSNP.json [hunk 10]
```diff
@@ -1,20 +1,75 @@
...
+  "name": "Join our Enthusiastic Open-source GrowInCommunity",
+      "url": "https://twitter.com/GrowInComm",
+      "icon": "FaLink"
+  },
+
+  {
```
Comment (priyankarpal) [inline data/RockingSNP.json]: Delete these duplicate lines from 71 to 102. You already added those before.

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by priyankarpal — APPROVED** [review]
LGTM

**Review by priyankarpal — DISMISSED** [review]

**Comment by RockingSNP** [comment]
Please merge this now

**Review by priyankarpal — APPROVED** [review]
LGTM 🎉 Maintanier will merge it soon.

File: data/RockingSNP.json [hunk 11]
```diff
@@ -1,48 +1,68 @@
...
-    }
+{
+      "group": "Community",
+      "name": "Join our Enthusiastic Open-source GrowInCommunity",
+      "url": "https://twitter.com/GrowInComm",
+      "icon": "FaLink"
+  }
+
   ]
+
```
Comment (krshkun) [inline data/RockingSNP.json:67]: Looks great, empty line will be fixed by gh actions

**Review by krshkun — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Great collaboration everyone. 
Looks good! 👍

**Comment by RockingSNP** [comment]
> Great collaboration everyone. 
> Looks good! 👍 

Thank you so much..

**Pull request merged**